rad/src/operators/string.rs

```diff
@@ -264,12 +280,62 @@ fn json_to_cbor(value: &json::JsonValue) -> Value {
     }
 }
 
+/// Replace thousands and decimals separators in a `RadonTypes` that is assumed to be a
+/// `RadonString`.
+///
+/// If the input value is not a `RadonString`, returns `RadError` because of `try_into`.
+pub fn replace_separators(
+    value: RadonTypes,
+    thousands_separator: serde_cbor::Value,
+    decimal_separator: serde_cbor::Value,
+) -> Result<RadonTypes, RadError> {
+    let rad_str_value: RadonString = value.try_into()?;
+
+    Ok(RadonTypes::from(RadonString::from(_replace_separators(
+        rad_str_value.value(),
+        thousands_separator,
+        decimal_separator,
+    ))))
+}
+
+/// Replace thousands and decimals separators in a `String`.
+pub fn _replace_separators(
+    value: String,
+    thousands_separator: serde_cbor::Value,
+    decimal_separator: serde_cbor::Value,
+) -> String {
+    let thousands = from_value::<String>(thousands_separator).unwrap_or_else(|_| ",".to_string());
+    let decimal = from_value::<String>(decimal_separator).unwrap_or_else(|_| ".".to_string());
+
+    value.replace(&thousands, "").replace(&decimal, ".")
+}
+
+/// Read separators from RAD call arguments, and fall back to the default ones if not provided.
+pub fn read_separators_from_args(
+    args: &[serde_cbor::Value],
+    skip: usize,
+) -> (serde_cbor::Value, serde_cbor::Value) {
```

Comment:
This function could return a pair of strings, because later in `_replace_separators` the `Value`s are converted to `Strings` anyway, and in case of error the result is set to the same default value as here.

```suggestion
) -> (String, String) {
```

Or a `Result<(String, String)>` if we wanted to handle errors, but I guess returning the default separators in case of error is fine.